pages/tutorials/parallel_builds.md.erb

```diff
@@ -1,6 +1,6 @@
 # Parallel Builds
 
-Running a build's jobs in parallel is a way to decrease your build's total running time. This guide will show you how to use multiple agents and job parallelism to increase the speed of your builds.
+Running a build's jobs in parallel is a way to decrease your build's total running time. This guide will show you how to use multiple agents and job parallelism to increase the speed of your builds. Note that parallelism for distinct steps is controlled primarily by [dependencies](/docs/pipelines/dependencies).
```

Comment:
I'm finding this a little confusing. Perhaps it could be good to set the stage a little?

```suggestion
Running a build's steps in parallel is a way to decrease your build's total running time. This guide will show you how to use multiple agents and job parallelism to increase the speed of your builds.

Steps run in parallel by default. If you define multiple steps, and [run multiple agents](#running-multiple-agents), the steps will run at the same time across your agents.

You can add [wait steps](...) or use [dependencies](...) to control when steps run in your pipeline.

Command steps can also be run in parallel, using [Parallel Jobs](#parallel-jobs).
```